README.md

```diff
@@ -4,18 +4,30 @@
 ![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/PorkStudios/FarPlaneTwo)
 ![Lines of code](https://img.shields.io/tokei/lines/github/PorkStudios/FarPlaneTwo)
 [![Discord](https://img.shields.io/discord/428813657816956929?color=7289DA&label=discord)](https://discord.gg/FrBHHCk)
-[![patreon badge](https://img.shields.io/badge/dynamic/json?color=e64413&label=patreon&query=data.attributes.patron_count&suffix=%20patrons&url=https%3A%2F%2Fwww.patreon.com%2Fapi%2Fcampaigns%2F727078)](https://www.patreon.com/DaPorkchop_)
+[![Patreon badge](https://img.shields.io/badge/dynamic/json?color=e64413&label=patreon&query=data.attributes.patron_count&suffix=%20patrons&url=https%3A%2F%2Fwww.patreon.com%2Fapi%2Fcampaigns%2F727078)](https://www.patreon.com/DaPorkchop_)
 
-[Download here](https://jenkins.daporkchop.net/job/PorkStudios/job/FarPlaneTwo/job/master/)
+*This is a Forge mod for Minecraft 1.12.2. Other versions WILL be supported in the future.*
 
-## This mod is work-in-progress!
+## Instructions
+1) Install [Minecraft Forge](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.12.2.html) for Minecraft 1.12.2.
+2) [Download](https://jenkins.daporkchop.net/job/PorkStudios/job/FarPlaneTwo/job/master/) the mod.
+3) Download these dependencies too:
+   - [ForgeRocks](https://www.curseforge.com/minecraft/mc-mods/forgerocks)
+   - [Mixin](https://www.curseforge.com/minecraft/mc-mods/mixin-0-7-0-8-compatibility)
+
+For best results use cubic chunks and cubic world gen and set the world type to custom cubic.<br>
+(Cubic Chunks bundles Mixin inside it, so don't use them both.)
+   - [Cubic Chunks](https://jenkins.daporkchop.net/job/OpenCubicChunks/job/CubicChunks/)
+   - [Cubic World Gen](https://jenkins.daporkchop.net/job/OpenCubicChunks/job/CubicWorldGen/)
+
+Some of the above pages should look a lot like this:<br> 
+![](download1.png)<br>
+There can be a lot of links on these pages. Click on the first one underneath "Last Successful Artifacts".
 
-*If you do decide to download this, don't expect stuff to work correctly because it probably won't.*
 
-**This mod currently supports the latest version of Forge for 1.12.2. It requires both [ForgeRocks](https://www.curseforge.com/minecraft/mc-mods/forgerocks) and [Mixin](https://www.curseforge.com/minecraft/mc-mods/mixin-0-7-0-8-compatibility) in order to launch correctly.**  
-(Mixin isn't needed if you're using [Cubic Chunks](https://github.com/OpenCubicChunks/CubicChunks))
 
 ## **[Read the FAQ first!](https://github.com/PorkStudios/FarPlaneTwo/wiki/FAQ)**
+### This mod is a work in progress!
```

Comment:
this belongs at the top, since the fact that the mod is work-in-progress is the first thing people should see